raiden_libs/messages/deserializer.py

```diff
@@ -1,11 +1,14 @@
 
+
 def deserialize(message: dict):
```

Comment:
One more thing: Is there a way to add new message types outside the libs module?

Reply:
in general all messages used in Raiden should be contained in the libs package, but we can make `type_to_class` public & extensible of course